static/src/js/util/url/collectionsEncoders.js

```diff
@@ -191,6 +209,12 @@ const decodedTemporalSubsetting = (pgParam) => {
   return enableTemporalSubsetting !== 'f'
 }
 
+const decodedSpatialSubsetting = (pgParam) => {
```

Comment:
You are missing tests for this, look in `projectCollections.test.js` for `describe('enable temporal subsetting flag', () => {`, should be the same tests for `ess`

Reply:
@macrouch Could you take a look again? I think I added conditions to account for it.

Reply:
Yes, it looks like you hit all the cases, but you should write new tests for this new functionality. Don't add additional complexity to the tests for unrelated functionality. When tests account for smaller chunks of functionality they are easier to understand and easier to maintain. 

I'm expecting to see `describe('enable spatial subsetting flag'``, with the same three test cases as the temporal subsetting tests